javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,14 @@
+/* exported findIndex */
+function findIndex(array, values) {
+  var count = 0;
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === values) {
+      return i;
+    } else {
+      count++;
+    }
+  }
+  if (count > 0) {
+    return -1;
+  }
```

Comment:
I'm not sure why this conditional would be necessary or the `count` variable, you can just `return -1` below your loop, because if the loop has ended, you know that the value was not in the array, or you would have returned the index in the loop and never reached this line of the code.